src/commands/login.rs

```diff
@@ -95,6 +95,14 @@ pub struct LoginCommand {
     )]
     pub hippo_password: Option<String>,
 
+    /// Auth Token
+    #[clap(
+        name = TOKEN,
+        long = "token",
+        env = TOKEN,
```

Comment:
We are generally lazy about this but I think this case merits decoupling the environment variable name from the argument name.  As @lann suggests, something like `SPIN_LOGIN_TOKEN` would be a good bet for the environment variable.  But in help (which uses the `name` by default) you probably want it to appear as `spin login --token <TOKEN>`.

Reply:
done